scripts/upload-apk.sh

```diff
@@ -51,15 +47,18 @@ if [ "$TRAVIS_BRANCH" == "$DEVELOPMENT_BRANCH" ]; then
     mv app-playstore-release.apk pslab-dev-release.apk
     mv app-fdroid-release.apk pslab-dev-release-fdroid.apk
     # Push generated apk files to apk branch
-    git checkout apk
-    git add -A
-    git commit -am "Travis build pushed [development]"
-    git push origin apk --force --quiet> /dev/null
 fi
+git checkout apk
+git add -A
+git commit -am "Travis build pushed [$TRAVIS_BRANCH]"
+
+git branch -D apk
```

Comment:
Cannot delete as it's already checked into apk branch

Reply:
Yeah, @neel1998 switch to temporary branch. I should have noticed it while reviewing, my bad